docs/docs/recipes/styling-css.md

```diff
@@ -240,7 +240,7 @@ Sass will compile `.scss` and `.sass` files to `.css` files for you, so you can
 plugins: [`gatsby-plugin-sass`],
 ```
 
-3. Write your stylesheets as `.sass` or `.scss` files and import them. If you don't know how to import styles, take a look at [Styling with CSS](/docs/recipes/#2-styling-with-css)
+3. Write your stylesheets as `.sass` or `.scss` files and import them. If you don't know how to import styles, take a look at [Styling with CSS](/docs/how-to/styling/built-in-css/)
```

Comment:
This is the only one I wasn't sure what to do with. I went with `/docs/how-to/styling/built-in-css/` because it immediately shows an example of importing a css file although I'm not sure how much more helpful it is since there's an example of importing the .sass/.scss file below and linking to an example involving a .css file may be misleading.

Another thought I had was maybe we don't need to link to a separate page here since there's an example of importing a little further below. Though, I'd probably split the JS example into two separate ones so that it doesn't give the impression that you'd write both the .sass and .scss files and them import them in the same JS file.

Reply:
I agree with you! I think the linked article goes into more detail and the examples below are a quick start reference. I would say both are good to keep.

As far as the examples here, I would lean toward either only using one example with scss (I like this less) OR splitting up the css/js blocks (I like this more), such as:

Using sass:
```
$font-stack:    Helvetica, sans-serif
$primary-color: #333
body
  font: 100% $font-stack
  color: $primary-color
```
```
import "./styles.sass"
```

Using scss:
```
$font-stack: Helvetica, sans-serif;
$primary-color: #333;
body {
  font: 100% $font-stack;
  color: $primary-color;
}
```
```
import "./styles.scss"
```

Along with (perhaps) moving the note from line 269 above these blocks.

I'd be ok with either option.

Reply:
Thanks for the feedback @j0sh77! I went ahead and split up the sass & scss examples